app/views/user_mailer/_due_today.html.erb

```diff
@@ -1 +1 @@
-<p><%= simple_format(replace_variables(@app_configs.upcoming_checkin_email_body)) %></p>
\ No newline at end of file
+<%= render html: simple_format(replace_variables(@app_configs.upcoming_checkin_email_body)).gsub('\n', '').html_safe %>

```

Comment:
This seems a bit excessive (the `gsub`) - we should just make sure that our default bodies don't have `\n` in them so that they don't show up for fresh instances. If an admin wants to put "\n" somewhere in their e-mails, we should let them.